sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvaluationSuite.scala

```diff
@@ -59,6 +60,10 @@ class ExpressionEvaluationBaseSuite extends FunSuite {
 
 class ExpressionEvaluationSuite extends ExpressionEvaluationBaseSuite {
 
+  def create_row(values: Array[Any]): Row = {
```

Comment:
`Row.apply` will call ScalaRefaction.convertToCatalyst(), we still need a wrapper to do that.